testtools/iothub_test/src/iothubtest.c

```diff
@@ -2008,3 +2010,23 @@ IOTHUB_TEST_CLIENT_RESULT IoTHubTest_SendMessage(IOTHUB_TEST_HANDLE devhubHandle
 
     return result;
 }
+
+const char* IoTHubTest_GetTargetNetworkInterface()
+{
+    const char* result = NULL;
+    static char networkInterface[32];
+    char* envVarValue = getenv("IOTHUB_TEST_TARGET_NETWORK_INTERFACE");
```

Comment:
nit: make the string a macro like other values in this file?

Reply:
Sounds like a good idea :)

Reply:
Done.